### PR TITLE
PouchDB!

### PR DESCRIPTION
**Pull request by gr2m**
Adaptions for https://github.com/hoodiehq/hoodie-server-store/pull/27. Needs review and split up of the commits, but it works :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,16 +36,15 @@ npm install
 
 There are a few options to change the behaviour of `hoodie-server`.
 
-option        | default                           |   description
-------------- | --------------------------------- | -------------
-path          | process.cwd()                     | Project path
```
Comment (christophwitzko) [inline README.md]: any particular reason why `options.path` got removed?

File: lib/config.js [hunk 2]
```diff
@@ -49,8 +48,9 @@ module.exports = function (options) {
       log.warn('config', 'Authentication details missing from database URL')
     }
   } else {
-    log.error('config', 'options.dbUrl must be set. PouchDB not supported at this point.')
-    // log.warn('config', 'No database URL provided, falling back to PouchDB')
+    log.info('config', 'No database URL provided, falling back to PouchDB')
+    config.db.prefix = '.hoodie/data/'
+    mkdirp.sync(projectPath + '/.hoodie/data')
```
Comment (christophwitzko) [inline lib/config.js]: always use `path.join` :wink:

File: lib/config.js [hunk 3]
```diff
@@ -49,8 +48,9 @@ module.exports = function (options) {
       log.warn('config', 'Authentication details missing from database URL')
     }
   } else {
-    log.error('config', 'options.dbUrl must be set. PouchDB not supported at this point.')
-    // log.warn('config', 'No database URL provided, falling back to PouchDB')
+    log.info('config', 'No database URL provided, falling back to PouchDB')
+    config.db.prefix = '.hoodie/data/'
```
Comment (christophwitzko) [inline lib/config.js]: I think it's better to use the absolute path here: `config.db.prefix = path.join(projectPath, '.hoodie/data/')`

File: README.md (hunk 1 above)
Comment (gr2m) [inline README.md]: I couldn’t find out where it was used. I _think_ we needed it to test hoodie-server with `my-first-hoodie`, but we don’t do that anymore. Or do we need it for another reason?

Comment (christophwitzko) [inline README.md]: It just adds the functionality that you can run hoodie-server outside your hoodie app folder. e.g.: `hoodie-server --path /my/hoodie/app`

Comment (gr2m) [inline README.md]: but why would we need that? I don’t see a use case for that?

**Comment by gr2m** [comment]
merging this as `tent` itself is a WIP branch :)

**Pull request merged**